doc-content/drools-docs/src/main/asciidoc/Examples/decision-examples-central-proc.adoc

```diff
@@ -8,9 +8,14 @@ You can use the example projects in {CENTRAL} to explore example business assets
 * {CENTRAL} is installed and running. For installation options, see {URL_PLANNING_INSTALL}[_{PLANNING_INSTALL}_].
 
 .Procedure
-. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click *Try Samples*.
+. Select *Menu* -> *Design* -> *Projects*.
+. Open the *MySpace* space.
+. Click *Try Samples*. If there are existing projects, you can access the samples by selecting *Try Samples* from the *Add Project* pull-down menu.
 +
-If a project already exists, click the three vertical dots in the upper-right corner of the *Projects* page and click *Try Samples*.
+
+.Try Samples
+image::dmn/try-samples.png[]
+
 . Review the descriptions for each sample project to determine which project you want to explore. Each sample project is designed differently to demonstrate
```

Comment:
I suggest that you undo your changes to this file, and then change only "If the project already exists, click the **Add Project** drop-down option in ..."  You can add that screenshot if you want, but I would avoid it IMO just to not add the vulnerability. And the way the first step is shown is correct as is for when a person doesn't have any projects. We don't need or want to mention going to MySpace because that might not be the case for all users, plus they actually bypass the MySpace page when they go to the projects page as instructed. And we always want to start BC instructions in modules with "In {CENTRAL}, ...". This was direct user feedback from a while ago, and makes sense. It's a usability concern, and is heightened by our mod doc approach.